Tasks/DeployVisualStudioTestAgent/task.loc.json

```diff
@@ -36,20 +38,20 @@
   ],
   "inputs": [
     {
-      "name": "testMachineGroup",
+      "name": "testMachines",
       "type": "multiLine",
-      "label": "ms-resource:loc.input.label.testMachineGroup",
+      "label": "ms-resource:loc.input.label.testMachines",
       "defaultValue": "",
       "required": true,
-      "helpMarkDown": "ms-resource:loc.input.help.testMachineGroup",
+      "helpMarkDown": "ms-resource:loc.input.help.testMachines",
       "groupName": "testMachineGroups"
```

Comment:
I kept *group because testMachine is group Name. not because of `testMachineGroup` ;)